DEPS.bzl

```diff
@@ -4132,8 +4132,8 @@ def go_deps():
         name = "com_github_tikv_client_go_v2",
         build_file_proto_mode = "disable_global",
         importpath = "github.com/tikv/client-go/v2",
-        sum = "h1:pfDrSVAnfkk2EkrOc0iOmtA4n8F6TL9oEAK8R/enC50=",
```

Comment:
Please fix the conflict.

Reply:
BTW, https://github.com/pingcap/tidb/pull/43225 is also updating the dependencies. Can you upgrade to the latest version?

Reply:
Upgraded by #43208